client/my-sites/plan-features-2023-grid/hooks/npm-ready/use-is-large-currency.ts

```diff
@@ -2,15 +2,36 @@ import { getCurrencyObject } from '@automattic/format-currency';
 import type { GridPlan } from 'calypso/my-sites/plan-features-2023-grid/hooks/npm-ready/data-store/use-grid-plans';
 
 const LARGE_CURRENCY_CHAR_THRESHOLD = 6;
+const LARGE_CURRENCY_COMBINED_CHAR_THRESHOLD = 9;
 
 interface Props {
 	gridPlans: GridPlan[];
 	returnMonthly?: boolean;
 }
 
+function formatPrice( price: number, currencyCode: string ) {
```

Comment:
FWIW, this function is now nearly exactly what `formatCurrency()` does, except that it doesn't take into account which side of the price the currency symbol is on (not that it probably matters in this context).

Reply:
Thanks for the heads up! I'll take a look in a few moments 👀

Reply:
Made the change in https://github.com/Automattic/wp-calypso/pull/81537/commits/82dd692cc4c549491e051fdbdd99f50b5dd7da30 😎